neutrino-custom/base.js

```diff
@@ -78,6 +80,10 @@ module.exports = neutrino => {
         .entry('failureviewer')
         .add(path.join(UI, 'entry-failureviewer.js'))
         .end();
+    neutrino.config
+        .entry('login')
+        .add(path.join(UI, 'entry-login.js'))
```

Comment:
Could you rename the file to `.jsx`? (#3163 will soon make ESLint check for this)

Reply:
@edmorley This is not the only react file. I assume you want me to change the auth callback pure react component to jsx files as well?

Reply:
Done